src/services/shims.ts

```diff
@@ -1109,6 +1122,12 @@ namespace ts {
                     info.compilerOptions);
             });
         }
+
+        public getSupportedCodeFixes(): string {

```

Comment:
this should not be needed now. if you need VS to call it, you need to expose this on the tsserver.